ha/core/system_health/model/entity_health.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import json
+
+from cortx.utils.conf_store import Conf
+from cortx.utils.log import Log
+from ha import const
+from ha.core.error import HaEntityHealthException
+
+class EntityEvent:
+    """
+    Entity Event. This class implements an entity event object,
+    which will be stored in the entity health.
+    """
+
+    def __init__(self, event_timestamp: str, created_timestamp: str, status: str, specific_info: dict=None):
+        """
+        Init method.
+        """
+
+        self.event_timestamp = event_timestamp
+        self.created_timestamp = created_timestamp
+        self.status = status
+        self.specific_info = specific_info
+
+    def ret_dict(self):
+        """
+        Return dictionary attribute.
+        """
+
+        return vars(self)
+
+class EntityAction:
+    """
+    This class implements an action object stored in the entity health.
+    """
+
+    def __init__(self, modified_timestamp: str, status: str):
+        """
+        Init method.
+        """
+        self.modified_timestamp = modified_timestamp
+        self.status = status
+
+    def ret_dict(self):
+        """
+        Return dictionary attribute.
+        """
+
```

Comment:
There are many unnecessary blank line in file
can we remove it

Reply:
Generally, I keep a blank line between method doc string and actual code inside the method. Also, I believe it is good to have blank lines inside method as well to separate code/increase the readability. Hence I would link keep these unless we there is a strong reason. Please let me know.

Reply:
`unless we there is a strong reason. Please let me know.`
There is no strong reason but some time codacy or some tool raise blank line error

Lets, keep it as here codacy not found any error.